atom/common/asar/archive.cc

```diff
@@ -107,6 +107,8 @@ bool FillFileInfoWithNode(Archive::FileInfo* info,
     return false;
   info->offset += header_size;
 
+  node->GetBoolean("executable", &info->executable);
```

Comment:
@zcbenz I see, so something like this should be enough, assuming `node->GetBoolean` will do nothing if the property is not defined, right?

```cpp
info->executable = false;
node->GetBoolean("executable", &info->executable);
```